lib/internal/cluster/master.js

```diff
@@ -41,15 +42,15 @@ if (schedulingPolicy === undefined) {
 
 cluster.schedulingPolicy = schedulingPolicy;
 
-cluster.setupMaster = function(options) {
+cluster.setupMaster = function(options = {}) {
```

Comment:
This is definitely a breaking change, albeit a subtle one. Setting an argument default necessarily makes this semver-major